src/main/java/Joey.java

```diff
@@ -0,0 +1,218 @@
+import java.util.Scanner;
+
+class Task {
```

Comment:
Should this be extracted out into its own class file? Likewise with all other non-main classes.